### ISSUE TITLE
Source map parsing fails when used in create-react-app typescript project

### ISSUE THREAD

**Issue by trombipeti** [mathuo/dockview#509]
**Describe the bug**
When using dockview in a create-react-app project with typescript, running "npm start" produces loads of warnings about not being able to parse the source map for dockview.

**To Reproduce**
```
npx create-react-app dockview-source-map-repro --template typescript
cd ./dockview-source-map-repro
npm install dockview
```
Replace code in App.tsx with the code from here: https://codesandbox.io/p/sandbox/github/mathuo/dockview/tree/master/packages/docs/sandboxes/simple-dockview?file=%2Fsrc%2Fapp.tsx%3A1%2C1-97%2C1
`npm start`

**Expected behavior**
I don't get loads of warnings

**Actual behavior**
Warnings like this are produced for all files in the dockview and dockview-core node modules:

```
WARNING in ./node_modules/dockview-core/dist/esm/api/component.api.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Work\playground\dockview-source-map-repro\node_modules\dockview-core\src\api\component.api.ts' file: Error: ENOENT: no such file or directory, open 'C:\Work\playground\dockview-source-map-repro\node_modules\dockview-core\src\api\component.api.ts'
```

**Comment by mathuo** [comment]
Thanks for raising this - yes the source maps are not configured correctly and do not work as expected.

I will remove them for now and re-visit added source maps later on if there is demand since configuring them correctly would require a few changes.

**Comment by trombipeti** [comment]
Thanks for the quick fix, I personally don't need the source maps so it's fine :)

**Comment by mathuo** [comment]
Released in 1.10.0 https://dockview.dev/
